dashboard/app/views/home/teacher_homepage.html.haml

```diff
@@ -1,4 +1,21 @@
+:ruby
 
-#container
+  courses = current_user.in_progress_and_completed_scripts.slice(0,2).map do |script|
+    script_id = script[:script_id]
+    script_name = Script.find_by_id(script_id)[:name]
+    {
+      id: script_id,
+      script_name: script_name,
+      name: data_t_suffix('script.name', script_name, 'title'),
+      description: data_t_suffix('script.name', script_name, 'description_short'),
+      link: script_url(script_id),
+      image: ""
+    }
```

Comment:
It's worth noting that these are called Course1/Course2/etc, these are actually scripts.

I think this is fine for now, and it's good to have some reason data, but this will end up changing some once my Course work is ready (getting closer!).